moovingon_ai/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "bb6dac8c-c4cd-414d-9b12-baef330a03a0",
+  "app_id": "moovingon-ai",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "moovingon.ai is a NOC orchestration and automation platform",
+    "title": "moovingon.ai",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "moovingon.ai Guidelines",
+        "image_url": "images/moovingon_ai-guidelines.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "moovingon.ai Events",
+        "image_url": "images/moovingon_ai-events.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "moovingon.ai Integrations",
+        "image_url": "images/moovingon.ai-integrations.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Notifications",
+      "Category::Incidents",
+      "Category::Event Management",
+      "Category::Alerting",
+      "Category::Issue Tracking",
+      "Offering::Integration",
+      "Submitted Data Type::Events",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
```

Comment:
Please add relevant tags from this [list](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags). This should include: 
      "Queried Data Type::<FILL IN>",
      "Submitted Data Type::<FILL IN>"

Reply:
Submitted Data Type was added.

about Queried Data Type, I'm just getting the list of monitors and there is no option for me to pick there.